scarb/src/core/package/mod.rs

```diff
@@ -67,6 +69,10 @@ impl Package {
             .ok_or_else(|| anyhow!("package `{self}` has no target `{kind}`"))
     }
 
+    pub fn fetch_profile(&self, profile: &SmolStr) -> Result<Profile> {
+        self.manifest.profiles.fetch_profile(profile)
+    }
```

Comment:
This operation doesn't make sense. Profile is just a handle. You ask for underlying representation, and get back the handle, which you can easily construct yourself. What is more needed here is `has_profile` operation I think